src/pages/signin/LoginForm.js

```diff
@@ -143,8 +144,8 @@ class LoginForm extends React.Component {
             return;
         }
 
-        const phoneLogin = LoginUtils.getPhoneNumberWithoutSpecialChars(login);
-        const isValidPhoneLogin = Str.isValidPhone(phoneLogin);
+        const phoneLogin = LoginUtils.appendCountryCode(LoginUtils.getPhoneNumberWithoutSpecialChars(login));
```

Comment:
Let's parse the number, if it's valid let's use the e164 format.
`Session.beginSignIn(parsedPhoneNumber.possible ? parsedPhoneNumber.number.e164 : login);`